public/pages/CreateDetector/components/ConfigureAlerts/components/AlertCondition/AlertConditionPanel.tsx

```diff
@@ -420,16 +423,29 @@ export default class AlertConditionPanel extends Component<
                 onChange={this.onNotificationChannelsChange}
                 singleSelection={{ asPlainText: true }}
                 onBlur={refreshNotificationChannels}
+                isDisabled={!hasNotificationPlugin}
               />
             </EuiFormRow>
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
-            <EuiButton href={NOTIFICATIONS_HREF} iconType={'popout'} target={'_blank'}>
+            <EuiButton
+              href={NOTIFICATIONS_HREF}
+              iconType={'popout'}
+              target={'_blank'}
+              isDisabled={!hasNotificationPlugin}
+            >
               Manage channels
             </EuiButton>
           </EuiFlexItem>
         </EuiFlexGroup>
 
+        {!hasNotificationPlugin && (
+          <>
+            <EuiSpacer size="m" />
```

Comment:
Should we add the spacer to the callout component itself?

Reply:
We can move it it's of course very easy, but personally I wouldn't.

This is not very important case, but its very good example what I was thinking few times in the past.
So, `<NotificationsCallOut>` should not be aware of the parent layout logic (means it should be able to "stretch" as much as parent want, and, generally speaking, should not add outer margins and leave it to the parent too).

I've lately started thinking of it as single responsibility principle no matter its just css/layout-ing. Let me know if you think its more clean way to move it down, no prob.

Reply:
As discussed offline, we can leave it as is